deploy/operator.yaml

```diff
@@ -19,8 +19,6 @@ spec:
       - command:
         - hyperconverged-cluster-operator
         env:
-        - name: KVM_EMULATION
```

Comment:
@arachmani it looks like this wasn't added to the CSV template.  Can you open a PR to add it?

Reply:
https://github.com/kubevirt/hyperconverged-cluster-operator/pull/343